debian/changelog

```diff
@@ -7,6 +7,7 @@ dell-recovery (1.66) UNRELEASED; urgency=medium
   [ Yuan-Chen Cheng ]
   * sort disk name by type and size.
   * properly show disk with rst attribute.
+  * skip this step if we run in automatic-oem-config mode.
```

Comment:
This commit message should be clearer what step is being skipped

Reply:
please check the new pushed branch, thanks.